titan-server/src/schema.rs

```diff
@@ -267,7 +288,22 @@ allow_tables_to_appear_in_same_query!(
 
 allow_tables_to_appear_in_same_query!(
     wcf1_user_to_group,
-    wcf1_acl_option_to_group
+    wcf1_acl_option_to_group,
+);
+
+allow_tables_to_appear_in_same_query!(
```

Comment:
This isn't unique to this PR, but I found out that you don't need to separate the tables that are allowed to appear in the same query. This macro just implements a trait for all of the tables listed, so we can combine all the tables into a single macro invocation instead of separating them. It also makes it easier to know which tables have had the trait implementation applied.

So, let's refactor all of the `allow_tables_to_appear_in_same_query!` macros into a single invocation.